### ISSUE TITLE
Intel Arc A770 SE(16GB) shows 0 GB of VRAM

### ISSUE THREAD

**Issue by just-Bri** [LiaNdrY/Enshrouded-Tool-Fix#4]
Unsure what else to post in terms of info or what could help.

I am on the latest driver, actually ran DDU in safe mode this morning and did a 100% clean install of the Arc drivers+software.

Code snippet of CLI output:
```bash
CPU Info...
Name CPU: 13th Gen Intel(R) Core(TM) i5-13600K
AVX support: Yes

Auto-selection of the paging file size: Yes

RAM: 32 GB

Video Card: Intel(R) Arc(TM) A770 Graphics (0 GB)
Attention: the amount of video memory is less than 6 GB
Additionally, with only 4GB of VRAM, the game limits texture settings. You can't change them in-game.
It is very likely that the game will show you a warning that your system does not meet the minimum requirements. You can acknowledge this message and proceed at your own risk.

The Texture Resolution setting affects the amount of video memory consumed by the game:
Performance (~5 GB), Balanced (~5.5 GB), Quality (~6.5 GB), Max.Quality (~8.5 GB)

It's recommended to update your video card drivers if you have an older version and a newer one is available.
After that, you must run this utility again to check the Vulkan API version.
```

**Comment by just-Bri** [comment]
After renaming `enshrouded_local.json` to `enshrouded_local.json.old` the game started up fine.  

I don't think this is really an "issue" because the game now works after running the tool, but reporting 0GB VRAM might scare some people

**Comment by LiaNdrY** [comment]
I didn’t have an Intel video card on hand, so I made the processing logic based on the Intel driver. Perhaps the standard settings differ from nVidia and AMD, although this is strange. In any case, in order to analyze the problem and find a solution, I need to remotely connect to the computer of the owner of the Intel graphics card to solve the problem.

**Comment by LiaNdrY** [comment]
@just-Bri You can export two branches in the registry to *.reg files and send them to me (if the files are large, you can archive them)

`Win + R, Regedit`

```
HKEY_LOCAL_MACHINE\HARDWARE\DEVICEMAP\VIDEO
HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Video
```

**Comment by just-Bri** [comment]
Here you go, if there's anything else I could send your way let me know.

[VideoReg.zip](https://github.com/LiaNdrY/Enshrouded-Tool-Fix/files/14971931/VideoReg.zip)

**Comment by LiaNdrY** [comment]
Thank you for the provided file, I will analyze it and report back as soon as there is any news.

**Comment by LiaNdrY** [comment]
Judging by the data provided, there is no way to determine the VRAM size of Intel video cards; the value that nVidia and AMD uses is ignored by Intel. I haven't found a solution to this problem yet.

**Comment by just-Bri** [comment]
@LiaNdrY you might want to post something over here: https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT

**Comment by LiaNdrY** [comment]
Can you export this thread?
\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\DirectX

**Comment by LiaNdrY** [comment]
@just-Bri Released version 1.1.3, try it, now everything should work.
